lib/services/nativescript-project-plugins-service.ts

```diff
@@ -179,6 +184,7 @@ export class NativeScriptProjectPluginsService implements IPluginsService {
 
 	public getPluginBasicInformation(pluginName: string): IBasicPluginInformation {
 		let [ name, version ] = pluginName.split("@");
+		name = name.toLowerCase();
```

Comment:
what will happen in case I want to add this package:
https://www.npmjs.com/package/NodObjC
**NOT** this one: https://www.npmjs.com/package/nodobjc
For example `$ appbuilder plugin add NodeObjC@1.0.0`